locksmith/src/utils/forage.ts

```diff
@@ -4,7 +4,7 @@ class Forage {
   }
 
   tokenMetadataDefaultImage(data: any) {
-    return `${this.normalizedAddress(data.address)}/metadata/default_image`
+    return `lock/${this.normalizedAddress(data.address)}/icon`
```

Comment:
I think we can revert this and the associated test as this code is in support of the lock owner provided default image

Reply:
ohhh, I was confused about that. This makes sense. I'll revert that bit.